about.html

```diff
@@ -0,0 +1,152 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/6b55e5dec8.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <title>HIV Global Summit 2022</title>
+</head>
+<body>
+<nav class="navigation-bar">
+    <a href="/" class="social__icons--link" target="_blank">
+        <i class="fab fa-twitter"></i>
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        <i class="fab fa-facebook"></i>
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        Kampala
+    </a>
+    <a href="/" class="social__icons--link" target="_blank">
+        Uganda
+    </a>
+</nav>
+<div class="container-1">
+<header>
+    <nav class="desktop-nav">
+        <div class="logo-image">
+            <img src="assets/images/download.png" alt="">
+            <a href="/index.html" class="logo-home">CC HIV Global Summit 2022</a>
+        </div>
+        <div>
+            <ul class="nav-header">
+                <li><a href="about.html" class="about" >About</a></li>
```

Comment:
- [ ] [OPTIONAL] Kindly change the link to redirect the user to the home page and the `<li>` tag text should be home, you are already on the about page so there is no reason to have a link for the same page here.